### DISCUSSION TITLE
Pinecone serverless support

### DISCUSSION THREAD

**Discussion by vkomenda** [FlowiseAI/Flowise#1568] [Q&A] [answered]
Pinecone serverless doesn't have an environment value. At the moment only Pinecone pods seem to be supported. Or is there a way to connect to serverless?

**Comment by HenryHengZJ** [comment]
PR in work - https://github.com/FlowiseAI/Flowise/pull/1561

**Reply by HenryHengZJ** [reply, marked as answer]
PR merged! You can now use both serverless and pod indexes as we got rid of pinecone the environment
